bitnami/contour/templates/daemonset.yaml

```diff
@@ -150,6 +151,7 @@ spec:
         - /config/envoy.json
         - --xds-address={{ template "contour.fullname" . }}
         - --xds-port=8001
+        - --resources-dir=/config/resources
```

Comment:
does this file exist? how are you creating it?

Reply:
I tested it like this.
`/config` is an emtpyDir and the config inside here is coming from the initcontainer I think.

```console
root@projectcontour-internal-envoy-vfvth:/# ls -la /config/resources/sds/
total 4
drwxr-x--- 2 1001 root 1024 Aug 10 20:24 .
drwxr-x--- 3 1001 root 1024 Aug 10 20:24 ..
-rw-r--r-- 1 1001 root  186 Aug 10 20:24 xds-tls-certicate.json
-rw-r--r-- 1 1001 root  185 Aug 10 20:24 xds-validation-context.json

root@projectcontour-internal-envoy-vfvth:/# mount |grep \/config
/dev/sda1 on /config type ext4 (rw,relatime)
```